pkg/crc/podman/podman.go

```diff
@@ -1 +1,96 @@
 package podman
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+const (
+	rootless = "crc"
+	rootful  = "crc-root"
+)
+
+type Config struct {
+	Runner               crcos.CommandRunner
+	PodmanExecutablePath string
+}
+
+func UsePodman() Config {
+	return Config{
+		Runner:               crcos.NewLocalCommandRunner(),
```

Comment:
Not sure why we need a runner, or to kind of implement the `Runner` interface in here? Looks like it would be good enough to use directly `RunWithDefaultLocale`? I don't mind using a `Runner`, but I'm not sure why we have public `RunPodmanCommand/RunPodmanCommandPrivate/...?